packages/react/src/use-chat.ts

```diff
@@ -513,12 +517,61 @@ By default, it's set to 0, which will disable the feature.
 
       event?.preventDefault?.();
 
+      const fileInput = event?.currentTarget.querySelector(
+        'input[type="file"]',
+      ) as HTMLInputElement;
+
+      let content: ContentPart[] = [];
+
+      if (input) {
+        content.push({
+          type: 'text',
+          text: input,
+        });
+      }
```

Comment:
do we have a similar stateless submit for regular input? if not i'm wondering if we should leave it out for now and only go for controlled inputs (which are best practice for react if i'm not mistaken)

Reply:
(we can always add something like this later in a separate change)

Reply:
`fileInput` here is used for clearing the input value, since they can't be mutated programmatically :/

will explore alternatives:
- using a ref for the file input
- resetting the form

Reply:
no longer relevant due to `files` parameter